.moai/specs/SPEC-EVAL-RUBRIC-001/research.md

```
@@ -0,0 +1,170 @@
+- **호환성**: ✅ 표준 관행. evaluator agent body에서 rubric 참조는 모범.
+- **권고 채택**: ACCEPT — 일반 도메인 evaluator-profiles에 mechanism 적용.
+
+---
+
+## 2. 현재 상태 (As-Is)
+
+### 2.1 default.md (이미 부분 구현)
+
+`.moai/config/evaluator-profiles/default.md` 분석:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)